src/test_driver/openiotsdk/integration-tests/common/fixtures.py

```diff
@@ -22,15 +22,43 @@
 
 import chip.CertificateAuthority
 import chip.native
-import pytest
 from chip import exceptions
 
-from .fvp_device import FvpDevice
-from .telnet_connection import TelnetConnection
+import pytest
+
+import pytest_asyncio
+
+import nest_asyncio
+
+from pyedmgr import TestCase, TestCaseContext, TestDevice
+
 from .terminal_device import TerminalDevice
 
 log = logging.getLogger(__name__)
 
+nest_asyncio.apply()
+
+__all__ = [
+    'rootDir',
+    'fvp',
+    'fvpConfig',
+    'telnetPort',
+    'networkInterface',
+    'otaProvider',
+    'softwareVersion',
+    'gdbPlugin',
+    'kvsStore',
+    'kvsFile',
+    'tdbStorageParam',
+    'psStorageParam',
+    'storageParam',
```

Comment:
I think these fixtures are unnecessary in the current version of integration tests. They are not passed in pytest calling.